Products/RhaptosModuleStorage/sql/db_schema.sql

```diff
@@ -242,6 +242,8 @@ CREATE UNIQUE INDEX modulekeywords_module_ident_keywordid_idx ON
 CREATE TABLE files (
     fileid serial PRIMARY KEY,
     md5 text,
+    sha1 text,
+    media_type text,
```

Comment:
media_type in the cnx-archive schema is the last declared value on the files table. I'm not sure this is problem a problem unless we are doing series inserts somewhere...